pyartifactory/models/Auth.py

```diff
@@ -16,3 +16,11 @@ class ApiKeyModel(BaseModel):
 
 class PasswordModel(BaseModel):
     password: SecretStr
+
+
+class AccessTokenModel(BaseModel):
+    access_token: str
+    expires_in: Optional[int] = 0
+    scope: Optional[str] = None

```

Comment:
I would rather define the scope variable as follows:

`scope: Dict[ScopeEnum, str]`